tide_core.install

```diff
@@ -1914,3 +1914,23 @@ function tide_core_update_8090() {
   $config_storage = \Drupal::service('config.storage');
   $config_storage->write($view, $source->read($view));
 }
+
+/**
+ * Remove deprecated modules before d10 upgrade.
+ */
+function tide_core_update_8091() {
+  $modules = [
+    'ckeditor',
+    'wysiwyg_template_core',
+    'color',
+    'hal',
+    'tide_spell_checker',
```

Comment:
`tide_spell_checker` Will not be supported anymore and the code will be cleaned up in the Drupal 10 upgrade